### ISSUE TITLE
Clicking PRINT in a new Saved Layout does not Automatically generate the Pic List files 

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#141]
Clicking PRINT should automatically create whatever is needed to accomplish the task. 
I have "Use Pic List to control pic size and location" checked.
I make other tweaks to the layout.
I save it to a new name.
I then exit
I then open it and click PRINT
Nothing happens. 
A file compare showed the new Layout folder did not have a Pic List folder in it.
Only after clicking Generate did PTXprint created the list of files needed. (But these files had errors in them! See another issue.)
It would be better if these file were generated automatically **the first time** a user clicks PRINT if there are no files in the layout's Pic List folder (and the user has specified that Pic List files should be used).
After the Pic List folder is created, clicking PRINT should NOT regenerate them.

**Comment by markpenny** [comment]
Good point. I'll see what I can do...

**Comment by markpenny** [comment]
should be working right in 0.8.8
